sw/device/silicon_creator/mask_rom/mock_sigverify_keys_ptrs.h

```diff
@@ -0,0 +1,41 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef OPENTITAN_SW_DEVICE_SILICON_CREATOR_MASK_ROM_MOCK_SIGVERIFY_KEYS_PTRS_H_
+#define OPENTITAN_SW_DEVICE_SILICON_CREATOR_MASK_ROM_MOCK_SIGVERIFY_KEYS_PTRS_H_
+
+#include "sw/device/lib/testing/mask_rom_test.h"
+#include "sw/device/silicon_creator/mask_rom/sigverify_keys_ptrs.h"
+
+namespace mask_rom_test {
+namespace internal {
+
+/**
+ * Mock class for sigverify_keys_ptrs.h
+ */
+class MockSigverifyKeysPtrs : public GlobalMock<MockSigverifyKeysPtrs> {
+ public:
+  MOCK_METHOD(const sigverify_mask_rom_key_t *, rsa_keys_ptr_get, ());
```

Comment:
nit: Should these functions use CamelCase naming to be C++ style compliant?

Reply:
Yes, done. Thanks!